packages/devextreme-react/tsconfig.json

```diff
@@ -3,7 +3,8 @@
 
   "compilerOptions": {
     "baseUrl": "./",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "downlevelIteration": true
```

Comment:
For what reason do we need this option?

Reply:
We need this option to be able to iterate maps ([here](https://github.com/DevExpress/DevExtreme/pull/25781/files#diff-8d058ac323cdd500661600d2ce8ec227cc7ede855a5246274af71525ec625dadR56)) while targeting an old ES.